src/main/resources/templates/typescript-language-server/clientSettings.json

```diff
@@ -3,6 +3,12 @@
   "completion": {
     "useContextAwareSorting": true
   },
+  "format": {
```

Comment:
I've only currently updated the TypeScript template. I can update other templates but would prefer to see which settings work best in each first, and I don't currently have anything other than TypeScript and CSS language servers available locally.